src/index.js

```diff
@@ -0,0 +1,58 @@
+import './index.css';
+// import { loadShows } from './modules/renderWindow.js';
+
+// loadShows();
+
+import displayShows from './modules/displayShows.js';
+import getShows from './modules/getShows.js';
+import commentPopup from './modules/commentPopup.js';
+import { getLikes } from './modules/displayLikes.js';
+import addLikes from './modules/addLikes.js';
+import getShowCount from './modules/showCounter.js';
+
+const showContainer = document.querySelector('.show-section');
+const popupContainer = document.querySelector('.popup-display');
+
+const addEvents = () => {
+  const likeIcons = document.querySelectorAll('.fa-heart');
+  likeIcons.forEach((likeIcon) => {
+    likeIcon.addEventListener('click', () => {
+      addLikes(likeIcon.dataset.id);
+    });
+  });
+};
+
+async function loadShows() {
+  const shows = await getShows();
+  displayShows(shows);
+  addEvents();
+  getLikes();
+
+  const showLink = document.querySelector('.show');
+  showLink.innerHTML = `Home (${getShowCount()})`;
+
+  document.addEventListener('click', async (e) => {
+    const commentButton = e.target.closest('.comment-btn');
+    const closeButton = e.target.closest('.close-icon');
+
+    if (commentButton) {
+      // eslint-disable-next-line
+      const selectedShow = shows.find((show) => show.id === parseInt(commentButton.id));
```

Comment:
- [OPTIONAL] The linters has the purpose of preventing bugs in your code and disabling them should be avoided in most cases. It would be nice if you try to solve them instead of disabling them.